deployment/template-job.yaml

```diff
@@ -121,13 +130,13 @@ parameters:
   description: Name of the bucket which contains the model
   required: true
   name: AWS_BUCKET_NAME_MODEL
-  value: "avgupta-dev-gokube-triage"
```

Comment:
So this won't be used after your PR #25 right? I can remove it after merging #25. But shouldn't keep inference result as a backup in S3 as well?

Reply:
We are already using it and reading details of s3 from Vault in your changes. So it will store inference outcome into s3 and read model from s3 bucket. Am I missing anything?

Reply:
yes, it does reads model files from S3 and writes inference result into different S3 bucket.

Reply:
I will remove it once your PR merged and I rebase with the master.